platform/openide.util.ui/src/org/netbeans/modules/openide/util/NbMutexEventProvider.java

```diff
@@ -24,9 +24,11 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 import org.openide.util.BaseUtilities;
+import org.openide.util.Lookup;
```

Comment:
Write a test for `Mutex.EVENT` interaction.

Reply:
@jtulach is this note still relevant after 1b68acdc17 ?